src/mud/examples/linear.py

```diff
@@ -800,13 +828,13 @@ def err(xs):
     _ = ax.set_ylabel("Relative Error")
     _ = ax.set_xlabel("Dimension of Output Space")
     _ = ax.legend(["MUD", "MAP", "Least Squares"])
-
-    save_figure(
-        "lin-dim-cov-convergence.png",
-        save_path=save_path,
-        dpi=dpi,
-        close_fig=close_fig,
-    )
+    if save_path:
```

Comment:
Should be 
`if save_path is not None:`
See [PEP 8](https://peps.python.org/pep-0008/)